.gitignore

```diff
@@ -165,3 +165,4 @@ gulp-cache
 .rollupcache
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31828&commitId=f563688fd909d72a9869aede4e10bda181e85032&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|5| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|0| Removed|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|0| Removed|
|Avatar Converged.basic - High Contrast.normal.chromium.png|0| Removed|

</details>


  <details><summary><b>SwatchPicker Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.Shape.default.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>